src/Microsoft.TemplateEngine.Cli/Microsoft/TemplateEngine/Utils/InstallRequestPathResolution.cs

```diff
@@ -42,7 +42,7 @@ private static IReadOnlyList<string> DetermineDirectoriesToScan(string baseDir,
         public static IEnumerable<string> Expand(string unexpandedInstallRequest, IEngineEnvironmentSettings environmentSettings)
         {
             // Handle cases like PackageId::Version
-            if (unexpandedInstallRequest.IndexOfAny(Path.GetInvalidFileNameChars()) != -1)
+            if (unexpandedInstallRequest.IndexOfAny(Path.GetInvalidPathChars()) != -1)
```

Comment:
Idea when I wrote this was... If user passes in `MyAwesomeTemplate::1.2.3.4` we want to bail out right away and not try to do any expansion... To achive that and cover some more cases I wanted to use `Path.GetInvalidPathChars()` but that didn't include `:`... Hence I switched to `Path.GetInvalidFileNameChars()` which includes `:`, I just realized that includes also `*` which makes this `if` very wrong....
So we have this options:
a) Remove comment above this `if` because its not true anymore
b) Remove whole if
c) Check just for `:`

Reply:
Issue with ':' is: C:\MyAwesomeTemplate is a valid input.

Reply:
Lul... Can you just remove comment above this if?